config/rbac/cluster/role.yaml

```diff
@@ -102,18 +102,40 @@ rules:
 - apiGroups:
   - postgres-operator.crunchydata.com
   resources:
-  - pgupgrades
+  - pgadmins
   verbs:
+  - create
+  - delete
   - get
   - list
+  - patch
+  - update
   - watch
 - apiGroups:
   - postgres-operator.crunchydata.com
   resources:
+  - pgadmins/finalizers
   - pgupgrades/finalizers
   - postgresclusters/finalizers
   verbs:
   - update
+- apiGroups:
+  - postgres-operator.crunchydata.com
+  resources:
+  - pgadmins/status
```

Comment:
wouldn't we just add `pgadmins/status` to the existing "status" block - similar to how `pgadmins/finalizers` :question: 

hmmm :thinking:  this is all generated right... so we don't have much control - nvm :upside_down_face: